stories/components/personCard.stories.js

```diff
@@ -21,22 +21,27 @@ export default {
   parameters: { options: { selectedPanel: 'storybookjs/knobs/panel' } }
 };
 
-export const personCard_hover = () => html`
-  <mgt-person person-query="me" show-name show-email person-card="hover"></mgt-person>
-
-  <div style="margin:2em 0 0 1em;font-family:segoe ui;color:#323130;font-size:12px">
-    (Hover on person to view Person Card)
-  </div>
+export const personCard = () => html`
```

Comment:
is this rename going to change the url and break our docs?

Reply:
Good catch. Maybe we should create aka links and update urls in aka links... Just a thought. Is it difficult to update links in docs in terms of process?

Reply:
would aka links work in the iframe? re updating docs, we just need to time it right so they get updated at the same time. @elisenyang, is it too late to change the url in the docs?

Reply:
@beth-panx @nmetulev does this need to be updated for 1.2?

Reply:
I have added a new one instead of changing this one. So we are good.